docs/cli/request-json/tools-composite.json

```diff
@@ -0,0 +1,6 @@
+[
+    {
+      "composite": {
+        }
+    }
+]
```

Comment:
```suggestion
]
```

Reply:
I'm trying to suggest that you add a newline to the end of the file, but if I suggest it, the commenting functionality removes it (probably considering it extraneous whitespace).